js/utils/Clipboard.js

```diff
@@ -23,7 +23,43 @@ define(['clipboard'], (clipboard) => {
 				console.log(e);
 			});
 		}
-	};
+
+
+        copyToClipboard(clipboardButtonId, clipboardButtonMessageId, appendedText) {
```

Comment:
Why did you leave previous `copyToClipboard` implementation? How could it be used? This is not Java